### DISCUSSION TITLE
I just wanted to say that fs.rename is not suitable for file movement

### DISCUSSION THREAD

**Discussion by moneyDev1111** [nodejs/help#5050] [Show and tell]
Instead 
```js
await copyFile(oldPath, newPath);
await unlink(oldPath);
```
should be used. Otherwise the output file might be blank :(((((((

**Comment by bnoordhuis** [comment]
...because?

**Reply by moneyDev1111** [reply]
Idk, I wanted to report an issue but since there's no issues tab, I decided to say it here

**Reply by bnoordhuis** [reply]
"it doesn't work" generally isn't very actionable as a bug report...

**Reply by moneyDev1111** [reply]
Go get some rest, you're hallucinating

**Reply by bnoordhuis** [reply]
Don't be a jerk, we don't tolerate those here.
